config/application.rb

```diff
@@ -90,15 +90,15 @@ class Application < Rails::Application
     # Customize any additional options below...
 
     config.autoload_paths += config.eager_load_paths
-    config.autoload_paths << Rails.root.join("app", "models", "aliases")
-    config.autoload_paths << Rails.root.join("app", "models", "mixins")
-    config.autoload_paths << Rails.root.join("lib", "miq_automation_engine", "models")
-    config.autoload_paths << Rails.root.join("lib", "miq_automation_engine", "models", "mixins")
-    config.autoload_paths << Rails.root.join("app", "controllers", "mixins")
-    config.autoload_paths << Rails.root.join("lib")
-    config.autoload_paths << Rails.root.join("lib", "services")
-
-    config.autoload_once_paths << Rails.root.join("lib", "vmdb", "console_methods.rb")
+    config.autoload_paths << Rails.root.join("app", "models", "aliases").to_s
+    config.autoload_paths << Rails.root.join("app", "models", "mixins").to_s
+    config.autoload_paths << Rails.root.join("lib", "miq_automation_engine", "models").to_s
+    config.autoload_paths << Rails.root.join("lib", "miq_automation_engine", "models", "mixins").to_s
+    config.autoload_paths << Rails.root.join("app", "controllers", "mixins").to_s
+    config.autoload_paths << Rails.root.join("lib").to_s
+    config.autoload_paths << Rails.root.join("lib", "services").to_s
+
+    config.autoload_once_paths << Rails.root.join("lib", "vmdb", "console_methods.rb").to_s
```

Comment:
we probably want to link to the ruby bug here so we can fix this hack when ruby 2.3/2.4 are fixed

Reply:
Agreed.  Good call.

Reply:
Going to merge anyway...the comment can be added when we add the spec we discussed in Gitter.